src/VisualStudio/Core/Test/Telemetry/VSTelemetryLoggerTest.vb

```diff
@@ -1,4 +1,5 @@
-﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+﻿#If False Then ' not supported for now
```

Comment:
@heejaechang Please skip this with proper Skip attributes pointing to a tracking bug.